satellite/server_register.yml

```diff
@@ -4,5 +4,7 @@
   vars:
     # env: undef
     satellite_url: "{{ lookup('ansible.builtin.env', 'SATELLITE_SERVER') }}"
+    cockpit_packages: minimal
   roles:
     - demo.satellite.register_host
+    - redhat.rhel_system_roles.cockpit
```

Comment:
I'd like to keep this separate from the registration process. Could you add a job template to linux/setup.yml that uses the existing system_roles playbook and specifies the role + variables in extra_vars on the job template?